monitor/alerts/list-get-example-warnings-apr01-apr30/list-get-example-warnings-apr01-apr30.3.x.js

```diff
@@ -0,0 +1,14 @@
+// Download the Node helper library from twilio.com/docs/node/install
+// These are your accountSid and authToken from https://www.twilio.com/console
+const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const authToken = 'your_auth_token';
+const params = {
+  startDate: '2015-04-01T00:00:00Z',
+  endDate: '2015-04-30T23:59:59Z',
+  logLevel: 'warning',
+};
+
+const client = require('twilio')(accountSid, authToken);
+
+client.monitor.v1.alerts.list(params)
```

Comment:
It says that theres no `client.monitor.v1.alerts.list(params).each()` function.
`client.monitor.v1.alerts.each()` function exists and works.
Just like that it works ok.